notes/stories/film/film-scifi-2000s.st.txt

```diff
@@ -3053,6 +3053,55 @@ brother and brother [John's older brother, Marciello, spoke to John over a cell
 coping with the death of a spouse [John wept while cradling the just died from exposure body of his wife, Elena.]
 
 
+movie: Undead (2003)
+====================
+
+:: Title
+Undead
+
+:: Description
+Undead is a 2003 Australian zombie science fiction comedy horror film written
+and directed by Michael and Peter Spierig and starring Felicity Mason, Mungo
+McKay and Rob Jenkins.
+
+Synopsis: The inhabitants of an Australian rural community begin turning into
+zombies.
+
+:: Date
+2003-02-27
+
+:: References
+https://en.wikipedia.org/wiki/Undead_(film)
+
+:: Ratings
+2 <paul>
+
+:: Choice Themes
+zombie [Most of the inhabitants of a rural community turned into mindless, brain-eating zombies after some strange meteors crashed down to the area from space.]
+human vs. zombie [A small group of survivors were terrorized by mindless, brain-eating zombies.]
+what if everyone turned into zombies [Zombiesm spread like a contagion through the inhabitants of a rural community.]
```

Comment:
Spelling 🧐
https://www.merriam-webster.com/dictionary/zombiism

Reply:
The situation is worse still. The phrase "Zombiism spread like a contagion through the inhabitants of a rural community." could well mean that the beliefs and practices of zombie cults could have been spreading through the community, which, IMHO, would have made a much more interesting movie.

A complete rewrite of the motivation has been effected.